conf/output/test-output-52.json

```diff
@@ -65,7 +65,7 @@
     "type" : [ "Contribution" ]
   }, {
     "agent" : {
-      "id" : "https://d-nb.info/gnd/VCN 77",
+      "id" : "VCN 77",
```

Comment:
This is the output for elasticsearch, right? It does not look correct that the `id` field is filled with a non-URI string. There is the same problem in conf/output/test-output-65.json.

Reply:
It's actually kind of a fix: if the value in the source field is not actually an ID (but a label), we no longer prepend the base URI. For consistency (to always have `id` and `label`), the label is used for the ID field. This might not be the final solution, but I think we can look into this when we tweak the web UI.